networkd/network_server.ml

```diff
@@ -373,7 +373,7 @@ module Interface = struct
 
 	let get_capabilities _ dbg ~name =
 		Debug.with_thread_associated dbg (fun () ->
-			Fcoe.get_capabilities name
+			(Fcoe.get_capabilities name) @ (SRIOV.get_capabilities name)
```

Comment:
I would recommend to use
```
List.concat
[ Fcoe.get_capabilities name
; SRIOV.get_capabilities name
]
```

This linear order makes it easier to add more such calls in the future.

Reply:
I agree. Thanks. Will change